tethys_cli/db_commands.py

```diff
@@ -380,6 +380,6 @@ def process_args(args):
 )
 
 
-def db_command(args):
+def db_command(args, unknown_args=""):
```

Comment:
Is `unknown_args` a string when it is given? If not, then you should set the default to `None` in all cases.

Reply:
It is a list actually. I've updated the code to None.